test/sparse/cholmod.jl

```diff
@@ -1,11 +1,13 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-srand(123)
-
 using Base.SparseArrays.CHOLMOD
 using DelimitedFiles
+using Test
+
+# CHOLMOD tests
+guardsrand(123) do
```

Comment:
Thanks for using this! but if I'm not mistaken, a global `srand(123)` does not leak outside of a test file, because there is an implicit `srand(global_seed)` before running each test files (if used via "test/runtests.jl") (unless maybe if the file is `include`d). So this `guardsrand` is maybe un-necessary here.